fundamentos/bloco-4-introducao-a-javascript-e-logica-de-programacao/dia-4-javascript-objetos-e-funcoes/exercicios/funcoes-3.js

```diff
@@ -0,0 +1,12 @@
+// 3 - Crie uma função que receba um array de inteiros e retorne o índice do menor valor.
+// Array de teste: [2, 4, 6, 7, 10, 0, -3]; .
+// Valor esperado no retorno da função: 6 .
+
+let array = [2, 4, 6, 7, 10, 0, -3];
+
+function lessValue(x) {
+
+    let lessNum = Math.min(...x)
```

Comment:
Isso vai retornar o menor valor, não o menor índice.
```suggestion
    let lessNum = Math.min(...x);
    return x.indexOf(lessNum);
```